xml/System.Security.Cryptography.Pkcs/Pkcs12SafeBag.xml

```diff
@@ -37,11 +37,24 @@
         <Parameter Name="skipCopy" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="bagIdValue">To be added.</param>
-        <param name="encodedBagValue">To be added.</param>
-        <param name="skipCopy">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="bagIdValue">The Object Identifier (OID), in dotted decimal form, indicating the data type of this SafeBag.</param>
+        <param name="encodedBagValue">The ASN.1 BER encoded value of the SafeBag contents.</param>
+        <param name="skipCopy"><see langword="true"/> to store <paramref name="encodedBagValue"/> without making a defensive copy; otherwise, <see langword="false"/>. The default is <see langword="false"/>.</param>
+        <summary>Called from constructors in derived classes to initialize the <see cref="T:System.Security.Cryptography.Pkcs12.Pkcs12SafeBag"/> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+## Remarks
+ The default behavior of this method is to make a defensive copy of `encodedBagValue`.
```

Comment:
I think the old text-to-markdown converter for the API introduced these spaces at the start of each line under ##remarks, but they aren't needed. Just FYI

Reply:
Well, that's good to know, I keep fighting with vim to leave them there.  Now I can stop doing that :)